pygeoapi/provider/oracle.py

```diff
@@ -67,16 +67,31 @@ def create_pool(cls, conn_dict, oracle_pool_min, oracle_pool_max):
            Lock is implemented before function call at __init__"""
         dsn = cls._make_dsn(conn_dict)
         # Create the pool
+        if conn_dict.get("external_auth") == "wallet":
```

Comment:
Suggest optimizing as follows:


```python

import logging as LOGGER


@classmethod
def create_pool(cls, conn_dict, oracle_pool_min, oracle_pool_max):
    """Initialize the connection pool for the class
       Lock is implemented before function call at __init__"""

    dsn = cls._make_dsn(conn_dict)

    connect_kwargs = {
        'dsn': dsn,
        'min': oracle_pool_min,
        'max': oracle_pool_max,
        'increment': 1
    }

    # Create the pool
    if conn_dict.get("external_auth", "") == "wallet":
        # If Auth is via Wallet you need to save a wallet under
        # the directory returned by this bash command if apache is used
        # cat /etc/passwd |grep apache
        # except another directory is specified in the sqlnet.ora file
        LOGGER.debug("Connection pool from wallet.")
        connect_kwargs["externalauth"] = True
        connect_kwargs["homegeneous"] = False

    else:
        LOGGER.debug("Connection pool from user and password.")
        connect_kwargs["user"] = conn_dict["user"]
        connect_kwargs["password"] = conn_dict["password"]

    p = oracledb.create_pool(**connect_kwargs)
    LOGGER.debug("Connection pool created successfully")

    return p

```

Reply:
Hi Tom, thank you for the feedback. I updated the provider with your feedback and added a brief reference to the docs for the configuration of wallet usage.